src/inverseRobot.js

```diff
@@ -2,12 +2,28 @@
 
 /*
  * @param {object} robot
- *
- * @return {object}
+*
+* @return {object}
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const newRobot = {};
+  let checkDoubleProper = false;
```

Comment:
The variable 'checkDoubleProper' is misleading and not needed. Its purpose is to check if there are duplicate properties, but this can be achieved without this variable.